README.md

```diff
@@ -20,21 +20,21 @@ To run a Docker container
 
   Linux agent:
 
-    docker run jenkins/jnlp-slave -url http://jenkins-server:port <secret> <agent name>
+    docker run --init jenkins/jnlp-slave -url http://jenkins-server:port <secret> <agent name>
 
   Windows agent:
 
-    docker run jenkins/jnlp-agent:latest-windows -Url http://jenkins-server:port -Secret <secret> -Name <agent name>
+    docker run --init jenkins/jnlp-agent:latest-windows -Url http://jenkins-server:port -Secret <secret> -Name <agent name>
```

Comment:
Does this work the same way on Windows? Also, it might be good to note what the --init flag does.

Reply:
I don't have Windows instance to check, so I just duplicated it from jenkinsci/docker-slave.
Docker docs don't specify `--init` behavior on Windows, but from sources, it seems ignored on Windows.